sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -2229,16 +2229,20 @@ public async Task StartRecognizeCustomFormsWithoutLabelsCanParseMultipageForm(bo
                     includeFieldElements: true,
                     expectedFirstPageNumber: expectedPageNumber,
                     expectedLastPageNumber: expectedPageNumber);
+            }
 
-                // Basic sanity test to make sure pages are ordered correctly.
-                var expectedLabelData = formIndex == 0 ? "__Address__1" : "Company Name:";
-                var expectedValueData = formIndex == 0 ? "Contoso Ltd. 2345 Dogwood Lane Birch, Kansas 98123" : "Southridge Video";
+            // Basic sanity test to make sure pages are ordered correctly.
 
-                FormField fieldInPage = recognizedForm.Fields.Values.Where(field => field.LabelData.Text.Contains(expectedLabelData)).FirstOrDefault();
-                Assert.IsNotNull(fieldInPage);
-                Assert.IsNotNull(fieldInPage.ValueData);
-                Assert.AreEqual(expectedValueData, fieldInPage.ValueData.Text);
-            }
+            FormPage firstFormPage = recognizedForms[0].Pages.Single();
+            FormTable firstFormTable = firstFormPage.Tables.Single();
+
+            Assert.True(firstFormTable.Cells.Any(c => c.Text == "Gold Sponsor"));
```

Comment:
The first page only has two fields: the one that changed behavior and another one that also appears in the second page. So we're using a table cell for differentiating them.